### ISSUE TITLE
[Accessbility]A11y_ Java Tooling for IntelliJ/Eclipse – New Feature_Spark Submission_Keyboard: Focus order is not proper in the ‘Spark Submission’ window

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3555]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #MAS2.4.3; #Keyboard; #C+E-H; #JavatoolingforIntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS2.4.3

**Repro Steps:** 
1) Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Open Eclipse and install the Scala plug-in 
4) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
5) In Azure Explorer section, tab till HDInsight and expand it. Select 'Spark on HDInsight (Scala)'. Tab till Next and press enter
6) Tab till Next button and press enter. Press Finish button
7) In Project Explorer, Expand the created Scala project right-click src, New-->Other and press enter
8) Tab till Scala Wizards, expand and select 'Scala Object' and press next
9) Enter the name of the file and press Finish button
10) Open Scala file and write any code in the editor
11) In Project Explorer view, right-click your project, choose Azure-->Submit Spark Application to HDInsight
12) Verify focus order is proper while navigating inside the window

**Actual Result:** 
Focus is going from the radio button to the ‘Cluster Name’ and then coming to the edit field present beside the radio button

**Expected Result:** 
Keyboard focus should navigate from top to bottom and from left to right. 
Here, focus can start from the ‘Cluster name’-->radio buttons-->fields present beside the label and to the controls present in the bottom

**User Impact:** 
Keyboard users will find difficulty in navigating if keyboard focus order is not proper in the window

**MAS Reference:** 
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

[Bug_1_Focus order is not proper in spark submission.pptx](https://github.com/microsoft/azure-tools-for-java/files/3635452/Bug_1_Focus.order.is.not.proper.in.spark.submission.pptx)

**Comment by t-rufang** [comment]
Note: Focus order is not proper. When focus is at radio button, click tab several times, focus flows in the following way.
![image](https://user-images.githubusercontent.com/32627233/65827364-fb0dda80-e2c3-11e9-8949-0b657026cce6.png)

**Comment by t-rufang** [comment]
Fixed in #3733

**Comment by Tanushagottipati** [comment]
Regressed in the below version and the issue is fixed. Hence closing the bug and attached latest screenshot for reference

Environment Details:
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.19.0

[Bug_3555_regressed_issue fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3863758/Bug_3555_regressed_issue.fixed.pptx)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749665
